easybuild/easyconfigs/g/GroIMP/GroIMP-1.5.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'GroIMP'
+version = '1.5'
+
+homepage = 'http://www.grogra.de/software/groimp'
+description = "GroIMP (Growth Grammar-related Interactive Modelling Platform) is a 3D-modelling platform."
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['GroIMP-%(version)s-src.zip']
+
+dependencies = [
+    ('Java', '1.8.0_112', '', True),
```

Comment:
What are these two additional fields? I've noticed them in other recent easyconfigs.

A new requirement?